generators/entity-server/templates/ui/widgets/tableWidget/src/i18n/locales.js.ejs

```diff
@@ -0,0 +1,7 @@
+import en from 'i18n/locales/en.json';
+import it from 'i18n/locales/it.json';
+
+export default {
```

Comment:
Probably this will need to be generated with the correspondent JHipster option, but is probably fine for now to keep it hardcoded